pkg/testing/integration/program.go

```diff
@@ -1710,13 +1732,34 @@ func (pt *ProgramTester) prepareGoProject(projinfo *engine.Projinfo) error {
 		return err
 	}
 
-	// skip building if the 'go run' invocation path is requested.
-	if !pt.opts.RunBuild {
-		return nil
+	// We only need to run dep-ensure if there is a Gopkg.toml file
+	_, err = os.Stat(filepath.Join(cwd, "Gopkg.toml"))
+	if err == nil {
+		err = pt.runCommand("dep-ensure", []string{goDepBin, "ensure", "-v"}, cwd)
+		if err != nil {
+			return fmt.Errorf("error installing go dependencies: %w", err)
+		}
 	}
 
+	// In our go tests, there seems to be an issue where we *need* to make a build before we
+	// actually run the tests. If a build isn't made, then we get an issue along the lines of
+	// `cannot import absolute path`. We will investigate this and add back in the respect
+	// for the runBuild option
```

Comment:
I’d love to identify what this issue is and open a bug to track it.

Reply:
Yeah I will open an issue to track this for sure